docs/index.html

```diff
@@ -36,540 +35,160 @@
 	<![endif]-->
 
 </head>
-<!-- BODY -->
-<!-- GRID-12 -->
```

Comment:
Comments are also nice lets keep those as well 👍. Remember this is for first time users.